### ISSUE TITLE
Auth error on billing screen

### ISSUE THREAD

**Issue by vydra** [airpair/airpair-com-issues#376]
I login with GitHub. When I go to Billing I see:

Authentication Error

Err... sorry about this. Please tell us what happened.

The balance seems correct.

**Comment by mjmj** [comment]
yeah I get the same thing, is airpair even maintained anymore? I get no response from Jonathon or support.

**Comment by vidkun** [comment]
Wow this is pretty old. I'm gonna guess that since there's no activity on it and I'm still getting the same error, @mjmj is probably correct that this is no longer maintained.

Any recommended replacements?

**Comment by mjmj** [comment]
Yeah and I have a balance of $200 paid into the site with no way to get my money or tutoring back. Really unfortunate.

**Comment by mjmj** [comment]
I do see some mentions of a newer airpair in open bugs so maybe they're just really bad at messaging?
